ndc-test/src/error.rs

```diff
@@ -5,7 +5,9 @@ pub type OtherError = Box<dyn std::error::Error + Send + Sync>;
 #[derive(Debug, thiserror::Error)]
 pub enum Error {
     #[error("error communicating with the connector: {0}")]
-    CommunicationError(#[from] super::client::Error),
+    CommunicationError(#[from] Box<super::client::Error>),
+    #[error("error communicating with the connector: {0} ")]
+    CommunicationErrorWithContent(Box<super::client::Error>),
```

Comment:
Why do we have both of these?

Reply:
No reason to have both of these, will remove the latter one.